athenahealth/resources/UpdateBookedAppointment_StringResponse.json

```diff
@@ -0,0 +1 @@
+"My name is Athena and I don't understand JSON types"
```

Comment:
Could we replace this with a realistic example of an error from this endpoint, like `"Invalid Appointment Status"` ?

Reply:
```
    main_test.go:394: 
        	Error Trace:	/home/userwork/code/github.com/CopilotIQ/spikes/ehr/athena/main_test.go:394
        	Error:      	Expected nil, but got: &errors.errorString{s:"Invalid PROVIDERID input"}
        	Test:       	TestNewPatientOnboarding
```

I'll drop in the error from my integration tests